andela/ah-backend-stark
# #161284824 cleanup migrations from develop

### What does this PR do?
It removes migration files from develop

### Description of Task to be completed?
Delete migration files from develop

### How should this be manually tested?
Checkout into the branch and explore the project's migrations folders.
To confirm that the server runs, go to the terminal and run:
`python manage.py makemigrations`
`python manage.py runserver`

### What are the relevant pivotal tracker stories?
#161284824
